Source/JavaScriptCore/runtime/JSIteratorConstructor.cpp

```diff
@@ -56,6 +56,9 @@ void JSIteratorConstructor::finishCreation(VM& vm, JSGlobalObject* globalObject,
     Base::finishCreation(vm, 0, vm.propertyNames->Iterator.string(), PropertyAdditionMode::WithoutStructureTransition);
     putDirectWithoutTransition(vm, vm.propertyNames->prototype, iteratorPrototype, static_cast<unsigned>(PropertyAttribute::DontEnum | PropertyAttribute::DontDelete | PropertyAttribute::ReadOnly));
     JSC_BUILTIN_FUNCTION_WITHOUT_TRANSITION(vm.propertyNames->from, jsIteratorConstructorFromCodeGenerator, static_cast<unsigned>(PropertyAttribute::DontEnum));
+
+    if (Options::useIteratorHelpers())
```

Comment:
Instead of using useIteratorHelpers, I think we should create a new feature flag option such as useIteratorSequencing.

Reply:
i can make a separate flag if you think it's needed but i felt it was close enough to the other iterator methods that it'd be find to group together

btw i think this was originally part of iterator helpers but later split out to discuss separately and it also relies on a few methods defined for iterator helpers, so it probably would have to be guarded by that flag anyways

Reply:
+1 on separate flag, we are about to enable `--useIteratorHelpers` by default